### PR TITLE
Microsoft.Data.Sqlite: Review resources

### PR DESCRIPTION
**Pull request by bricelam**
Part of #7201

/cc @Pilchie FYI

### CODE REVIEW

File: src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx [hunk 1]
```diff
@@ -130,16 +130,16 @@
...
-    <value>The IsolationLevel '{isolationLevel}' is invalid.</value>
+    <value>The IsolationLevel '{isolationLevel}' is not supported.</value>
   </data>
   <data name="InvalidParameterDirection" xml:space="preserve">
-    <value>The ParameterDirection '{direction}' is invalid.</value>
+    <value>The ParameterDirection '{direction}' is not supported.</value>
   </data>
   <data name="KeywordNotSupported" xml:space="preserve">
-    <value>Keyword not supported: '{keyword}'.</value>
+    <value>Connection string keyword '{keyword}' is not supported. For a possible alternative, see https://docs.microsoft.com/dotnet/standard/data/sqlite/compare#connection-strings</value>
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx:142]: @JeremyLikness Can I get a fwlink?

Comment (smitpatel) [reply src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx:142]: https://go.microsoft.com/fwlink/?linkid=2142181

Comment (JeremyLikness) [reply src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx:142]: @bricelam did @smitpatel give you what's needed or is there still action to take?

**Review by smitpatel — APPROVED** [review]

File: src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx (hunk 1 above)
Comment (bricelam) [reply src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx:142]: That depends, who's the owner of that fwlink? 😏 Don't we have a team SG we can use?

Comment (smitpatel) [reply src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx]: I add both of them as owners and have created 5 so far.

Comment (bricelam) [reply src/Microsoft.Data.Sqlite.Core/Properties/Resources.resx]: Sounds reasonable. I will do the same, thanks!

**Pull request merged**